crossbuilder

```diff
@@ -671,8 +751,22 @@ check_for_container_network() {
         sleep 1
     done
     if [ $NETWORK_UP -ne 1 ] ; then
-        echo -e "${ERROR_COLOR}Container is not connected to the Internet.${NC}"
-        exit 1
+        # On manjaro the 16.04 containers can't get ip4 address.
+        # This can be corrected by running 'dhclient' in container.
+        exec_container_root "dhclient"
+        for i in `seq 1 10`
+        do
+            if lxc query "/1.0/containers/${LXD_CONTAINER}/state" | jq -e '.network.eth0.addresses | any( .family == "inet" )' > /dev/null 2>&1 ; then
```

Comment:
Since this is the same command we have above, I suggest:

1. renaming this function to `ensure_container_network()`
2. create a small `check_container_network()` function containing the `lxc query command`